tendrl/monitoring_integration/objects/config/__init__.py

```diff
@@ -0,0 +1,16 @@
+from tendrl.commons import config as cmn_config
+from tendrl.commons import objects
+
+
+class Config(objects.BaseObject):
+    internal = True
+
+    def __init__(self, config=None, *args, **kwargs):
+        self._defs = {}
+        super(Config, self).__init__(*args, **kwargs)
+
+        self.data = config or cmn_config.load_config(
+            'monitoring-integration',
+            "/etc/tendrl/node-agent/node-agent.conf.yaml"
```

Comment:
/etc/tendrl/monitoring-integration/monitoring-integration.conf.yaml and define this sample conf file in etc/tendrl/monitoring-integration

similar to https://github.com/Tendrl/gluster-integration/blob/master/etc/tendrl/gluster-integration/gluster-integration.conf.yaml.sample